m0t1fs/linux_kernel/file.c

```diff
@@ -3348,6 +3348,42 @@ static void dgmode_rwvec_dealloc_fini(struct dgmode_rwvec *dg)
 	m0_varr_fini(&dg->dr_pageattrs);
 }
 
+static void databufs_set_dgw_mode(struct pargrp_iomap *iomap,
+				  struct m0_ext *ext)
+{
+	uint32_t         row_start;
+	uint32_t         row_end;
+	uint32_t         row;
+	uint32_t         col;
+	struct data_buf *dbuf;
+
+	page_pos_get(iomap, ext->e_start, &row_start, &col);
+	page_pos_get(iomap, ext->e_end - 1, &row_end, &col);
```

Comment:
The result "col" in this line and previous line, will have the same value?

Reply:
I believe so, yes.